README.md

```diff
@@ -165,3 +166,38 @@ tkn pipeline start operator-hosted-pipeline \
   --workspace name=registry-credentials,secret=my-registry-secret \
   --showlog
 ```
+
+
+## Operator Release pipeline
+The Release pipeline runs after the layers of validation (CI (optionally) and Hosted pipeline).
+It is used to certify and publish submitted bundle version.
+It is triggered by merging the submission pull request and sucessfully completes with updating the GitHub
+indicating published versions.
+
+### Installation
+
+```bash
+oc apply -R -f ansible/roles/operator-pipeline/templates/openshift/pipelines/operator-release-pipeline.yml
+oc apply -R -f ansible/roles/operator-pipeline/templates/openshift/tasks
+
+# Install external dependencies
+curl https://raw.githubusercontent.com/tektoncd/catalog/main/task/git-clone/0.4/git-clone.yaml | oc apply -f -
+```
+
+### Execution
+The release pipeline can be triggered using the tkn CLI like so:
+
+
+tkn pipeline start operator-release-pipeline \
+  --param git_repo_url=git@github.com:redhat-openshift-ecosystem/operator-pipelines-test.git \
+  --param git_revision=main \
+  --param bundle_path=operators/kogito-operator/1.6.0-ok \
+  --param imagestream=operator-pipelines \
+  --param tag=1.6.0-ok \
```

Comment:
Do we really need this param or could it be optional? By default I think it can come from the bundle.